maps/DeadSpace/Aegis_areas.dm

```diff
@@ -4,41 +4,53 @@
 	name = "\improper Aegis VII"
 	icon = 'maps/DeadSpace/ishimura_areas.dmi'
 	icon_state = "aegis"
+	sound_env = ASTEROID
```

Comment:
While we've not gotten around to fixing this yed, aegis VII is not an asteroid. It's not meant to be the lightless, airless rock it currently is. Aegis VII in canon is a rocky desert planet with a breathable atmosphere and a harsh noonday sun

Reply:
This is for a sound environment. It's for the reverb and sound reflection to take effect.

This is simply a very, very large open area basically. So, the issue you see is a non-issue.